custom_components/tuya_local/devices/klarstein_dryfy_pro_connect_dehumidifier.yaml

```diff
@@ -119,9 +119,9 @@ secondary_entities:
         optional: true
         mapping:
```

Comment:
You can remove the mapping entirely if the values do not need to be mapped to different ones.  I'm not sure why the original author of the config switched them like this, it does seem strange.